configuration/concepts/vaccination.csv

```diff
@@ -14,4 +14,6 @@ dd07efeb-47f1-4b72-9584-0152837b1b3b,,,HPV,VPH,HPV,VPH,,Misc,Text,,,,,,,,,,,
 43c8df3c-6c4c-4921-ba7c-a7d0f9dba4db,,,Meningococcal,Meningocócica,Meningococcal,Meningocócica,,Misc,Text,,,,,,,,,,,
 a415ac86-04d5-4e7b-a19b-c4abd431d930,,,HepB,HepB,HepB,HepB,,Misc,Text,,,,,,,,,,,
 018bc554-4d36-4887-a387-fb1b8bb20216,,,Influenza,Influenza,Influenza,Influenza,,Misc,Text,,,,,,,,,,,
-3c25e650-ccfa-48da-b276-a7872796b113,,,Vaccination History,Historial de vacunación,Vaccination History,Historial de vacunación,Vaccination History,ConvSet,N/A,,,Rotavirus; dT; Rougeole/Rubeole; Pentavalent; Polio oral; Pneumocoque; BCG; MMR; Var; HepA; Tdap; HPV; Meningococcal; HepB; Influenza,,,,,,,,
\ No newline at end of file
+2258b601-0c96-44c7-adb9-9d14d290da62,,,Covid,COVID-19,Covid,COVID-19,,Misc,Text,,,,,,,,,,,
```

Comment:
Use the same casing as "COVID-19" instead of Covid

Reply:
Done